apps/shooting-clubs/controllers/confirm.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../common/controllers/confirm');
+
+module.exports = class ConfirmController extends Controller {
+
+  getFieldData(field, req) {
+    const defaults = super.getFieldData(field, req);
+
+    if (field.field === 'location-addresses') {
+      return defaults.value.map(address => {
+        address['address-category'] = [].concat(address['address-category']);
```

Comment:
I tried writing a formatter to do this, then noticed that even if I do, `hof-for-controller` immediately slaps be back down - https://github.com/UKHomeOfficeForms/hof-form-controller/blob/master/lib/formatting/index.js#L27-L29